backend/siarnaq/api/episodes/models.py

```diff
@@ -259,6 +264,14 @@ class Tournament(models.Model):
     in_progress = models.BooleanField(default=False)
     """Whether the tournament is currently being run on the Saturn compute cluster."""
 
+    challonge_id_private = models.SlugField(null=True, blank=True)
+    """The Challonge ID of the associated private bracket."""
+
+    challonge_id_public = models.SlugField(null=True, blank=True)
+    """The Challonge ID of the associated private bracket."""
+
+    # TODO drop the following two fields when ready to make incompatible migrations
```

Comment:
Better now than later.

Reply:
jk, we're saving breaking migrations for once this PR is as ready as can be